multipatch_analysis/connection_detection.py

```diff
@@ -501,7 +501,16 @@ def fit_psp(self, **kwds):
         return fit_psp(response, **kwds)
 
 
-def fit_psp(response, mode='ic', sign='any', xoffset=11e-3, yoffset=(0, 'fixed'), mask_stim_artifact=True, method='leastsq', fit_kws=None, **kwds):
+def fit_psp(response, 
+            mode='ic', 
+            sign='any', 
+            xoffset=11e-3, 
+            yoffset=(0, 'fixed'), 
+            mask_stim_artifact=True, 
+            method='leastsq', 
+            fit_kws=None, 
+            stacked = False,
```

Comment:
Don't change the default; this is used in many places, some of which require stackedpsp.

Reply:
(or if you want to change the default, then you also have to change all the other places in the library that depend on this)